examples/extended_validation/src/tests.rs

```diff
@@ -0,0 +1,89 @@
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::{ContentType, Status};
+
+use super::rocket;
+
+fn test_login(username: &str,
```

Comment:
Sure, I'll use some shorter name and see it can fit into one line perfectly.

This code format was originally generated by the rustfmt. 
One small question, is there any recommend rust format or more properly coding style for this project.